src/search/pdbs/pattern_generator_rcg.h

```diff
@@ -0,0 +1,31 @@
+#ifndef PDBS_PATTERN_GENERATOR_RCG_H
+#define PDBS_PATTERN_GENERATOR_RCG_H
+
+#include "pattern_generator.h"
+
+namespace options {
+class Options;
+}
+
+namespace utils {
+class RandomNumberGenerator;
+enum class Verbosity;
+}
+
+namespace pdbs {
+class PatternGeneratorRCG : public PatternGenerator {
+    const int max_pdb_size;
+    const int max_time;
+    const bool bidirectional;
+    const utils::Verbosity verbosity;
+    std::shared_ptr<utils::RandomNumberGenerator> rng;
+public:
+    explicit PatternGeneratorRCG(options::Options &opts);
+    virtual ~PatternGeneratorRCG() = default;
```

Comment:
Isn't our policy to leave out the destructor if it has a default implementation?